conf.py

```diff
@@ -14,35 +17,24 @@
 templates_path = ['_templates']
 source_suffix = '.rst'
 master_doc = 'index'
-language = None
+language = 'en'
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', '**.ipynb_checkpoints', 'scanpy_workshop/*']
 pygments_style = 'sphinx'
 
 # -- Options for HTML output ----------------------------------------------
 
-html_theme = 'sphinx_rtd_theme'
-html_theme_options = dict(
-    navigation_depth=4, logo_only=True  # Only show the logo
-)
-html_context = dict(
-    display_github=True,      # Integrate GitHub
-    github_user='theislab',   # Username
-    github_repo='scanpy-tutorials',     # Repo name
-    github_version='master',  # Version
-    conf_py_path='/',    # Path in the checkout to the docs root
+html_theme = 'sphinx_book_theme'
+html_theme_options = dict( 
+    repository_url='https://github.com/theislab/scanpy-tutorials',
+    repository_branch='master',
+    use_repository_button=True,
 )
 html_static_path = ['_static']
-html_show_sphinx = False
 html_logo = '_static/img/Scanpy_Logo_BrightFG.svg'
 
-def setup(app):
-    app.add_stylesheet('css/custom.css')
-
 # -- Strip output ----------------------------------------------
 
-import nbclean, glob
-
-for filename in glob.glob('**/*.ipynb', recursive=True):
-    ntbk = nbclean.NotebookCleaner(filename)
+for path in Path().rglob('**/*.ipynb'):
+    ntbk = NotebookCleaner(str(path))
     ntbk.clear('stderr')
-    ntbk.save(filename)
+    ntbk.save(path)
```

Comment:
Why are we doing this? I thought the documentation is generated from notebooks including their outputs?

Reply:
I don’t know! Maybe to prettify them by having no red stderr blocks?

Reply:
I see, makes sense!